lib/mayaUsd/fileio/jobContextRegistry.h

```diff
@@ -75,6 +75,21 @@ class UsdMayaJobContextRegistry : public TfWeakBase
     /// Enabler function, returns a dictionary containing all the options for the context.
     typedef std::function<VtDictionary()> EnablerFn;
 
+    // Show a UI, a modal dialog, for the plugin options.
```

Comment:
Is this the option dialog for each entry that needs it?
If so, does it have to be modal?

Reply:
Yes, the designer would prefer that it be modal.